contribute/labels-of-issues.md

```diff
@@ -0,0 +1,116 @@
+# List of labels for issues and PRs on OpenEBS Project
+
+This document provides a list of labels and their purpose. These labels can be attached to issues and PRs. The aim behind having such list of labels is, we can sort and distinguish issues. This list will help new community members to dive-in OpenEBS if they wish to contribute to project or investigate the problem.
+
+Use case:
+> I am a beginner, I want to contribute to OpenEBS project and I am not sure from where to begin?
+
+Answer:
+> Start looking into the issues tagged with labels such as `help/small` , `size/XS`, `kind/unit-test` etc. Solve such issues and raise PR.
+
+---
+
+## Kind
+
+Labels prefixed with *kind* indicate the type of issue and PR. Suppose, you want to report a bug then you can tag your issue with `kind/bug` label.
+
+| Label | Description |
+|---|---|
+| kind/api  | Issue/PR related to API  |
+| kind/bug |  If there’s an error, failure in workflow, fault etc.|
+| kind/feature  | New / existing / required feature or enhancement |
+| kind/improve | Needs improvement |
+| kind/e2e-test | Need/Add end to end test |
+| kind/unit-test | Need/Add Unit test|
+| kind/security  | Security-related concerns  |
+| kind/testing  |  Needs/Adds tests |
+| kind/good-first  |  beginner friendly |
```

Comment:
Are we not using labels similar to https://github.com/openebs/mayastor/labels?

Reply:
Let's leave out the label discussion out of this PR. I will raise a different one once we all decide.